mp/game/momentum/resource/ui/Spectator.res

```diff
@@ -32,60 +32,61 @@
     {
         "ControlName"		"Label"
         "fieldName"			"PlayerLabel"
-        "xpos"				"10"
-        "ypos"				"20"
-        "wide"				"81"
-        "tall"				"15"
+        "xpos"				"cs-0.5"
+        "ypos"				"16"
+        "wide"				"160"
+        "tall"				"22"
         "autoResize"		"0"
         "pinCorner"			"0"
         "visible"			"1"
         "enabled"			"1"
-        "textAlignment"		"west"
         "dulltext"			"0"
         "brighttext"		"0"
-        "auto_wide_tocontents" "1"
+		"font"				"BigNoodleSmall"
```

Comment:
Convert the spacing of this file to spaces (in a new commit) to be consistent with the file.